.github/workflows/ci-workflow.yml

```diff
@@ -0,0 +1,73 @@
+name: ciworkflow
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!master'
+
+env:
+  # Setting an environment variable with the value of a configuration variable
+  ACCOUNT_ID: ${{ secrets.ACCOUNT_ID }}
+  API_KEY: ${{ secrets.API_KEY }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 7.x.x
+      - name: Install unzip
+        run: |
+          sudo apt-get update  
+          sudo apt-get install -y unzip
+      - name: Install OpenSSH and Git
```

Comment:
Do we really to install this since the build runs on `ubuntu-latest`? Is the package already installed? Can you try removing this?